Docs: minimal docs for links preview addons (#11828)
readthedocs/readthedocs.org: docs/user/links-preview.rst

> Links preview
> =============

Reviewer: ```suggestion
Link previews
=============
```

Author: Ouch, I've been using "Links preview" everywhere 🙃

> Links previews allows you to see the content of the link you are about to click when you hover it.
> This content will be rendered inside a popup that disappears after moving the mouse outside of it.

Reviewer: ```suggestion
Link previews allows you to see the content of the link you are about to click when you hover over it.
This gives you the ability to quickly preview where a link points,
so you can keep your context while reading but learn more from other resources. 
```

I think it's useful to give the "why should I care?" to the user in the first couple sentences.

.. figure:: /_static/images/addons-links-preview.png
   :width: 80%

>    Example of links preview addon

Reviewer: ```suggestion
   Example of link previews addon
```

> Enabling links previews
> -----------------------

Reviewer: ```suggestion
Enabling link previews
----------------------
```

> In your dashboard, you can go to the links preview tag in :guilabel:`Settings > Addons > Links preview` and enable it.

Reviewer: ```suggestion
In your dashboard, you can go to the links preview tag in :guilabel:`Settings > Addons > Link previews` and enable it.
```

> Troubleshooting links previews
> ------------------------------

Reviewer: ```suggestion
Troubleshooting link previews
-----------------------------
```
